src/prop-types/function.ts

```diff
@@ -23,6 +24,13 @@ export const functionProp = <T extends Function>(validator?: Validator): Functio
     default: undefined,
     validator: vuePropValidator(validator),
   },
+  nullable: {
+    type: Function,
+    required: false,
+    // eslint-disable-next-line unicorn/no-null
```

Comment:
I think we can disable the rule globally

Reply:
Done now in https://github.com/FloEdelmann/vue-ts-types/pull/89/commits/433f10686e1d9fd72508e4049d6cce21f7ad0308